lib/blueprinter/configuration.rb

```diff
@@ -24,18 +37,15 @@ def initialize
       @extractor_default = AutoExtractor
       @default_transformers = []
       @custom_array_like_classes = []
-    end
-
-    def extensions
-      @extensions ||= Extensions.new
+      @extensions = Extensions.new
```

Comment:
Doesn't this mean, in the code below, that `ExtensionA` would be dropped?

```ruby
Blueprinter.configure do |config|
  config.extensions << ExtensionA.new
  config.extensions << ExtensionB.new
end
```

Reply:
This should still work!

```ruby
irb(main):001> class ExtensionA; end
=> nil
irb(main):002> class ExtensionB; end
=> nil
irb(main):003* Blueprinter.configure do |config|
irb(main):004*   config.extensions << ExtensionA.new
irb(main):005*   config.extensions << ExtensionB.new
irb(main):006> end
=> #<Blueprinter::Extensions:0x000000014d03fa68 @extensions=[#<ExtensionA:0x000000014d03f978>, #<ExtensionB:0x000000014d03f8b0>]>
irb(main):007> Blueprinter.configuration.extensions
=> #<Blueprinter::Extensions:0x000000014d03fa68 @extensions=[#<ExtensionA:0x000000014d03f978>, #<ExtensionB:0x000000014d03f8b0>]>
irb(main):008> 
```

It's effectively the same behavior as before, just with the `@extensions` being set to `Extensions.new` _before_ the block is evaluated instead of on first `Configuration#extensions` call.